widgetssdk/src/main/java/com/glia/widgets/launcher/EngagementLauncher.kt

```diff
@@ -0,0 +1,57 @@
+package com.glia.widgets.launcher
+
+import android.app.Activity
+import com.glia.androidsdk.Engagement
+
+/**
+ * An interface for launching different types of engagements, such as chat,
+ * audio calls, video calls, and secure conversations
+ */
+interface EngagementLauncher {
+    /**
+     * Starts a chat engagement.
+     *
+     * @param activity The Activity used to launch the chat screen
+     */
+    fun startChat(activity: Activity)
+
+    /**
+     * Starts an audio engagement.
+     *
+     * @param activity The Activity used to launch the call screen
+     */
+    fun startAudioCall(activity: Activity)
+
+    /**
+     * Starts a video engagement.
+     *
+     * @param activity The Activity used to launch the call screen
+     */
+    fun startVideoCall(activity: Activity)
+
+    /**
+     * Starts a secure conversations engagement.
+     *
+     * @param activity The Activity used to launch the secure conversations screen
```

Comment:
NIT: if I remember correctly, the latest approved name is secure messaging, isn't it?

Reply:
Yep, somehow, I missed it here.

Reply:
Fixed!